cogs/TwitchAlert.py

```diff
@@ -749,8 +756,7 @@ async def get_team_users(self, team_id):
         :return: the JSON information of the users
         """
         url = 'https://api.twitch.tv/helix/teams?name=' + team_id
-        return (
-            await self.requests_get(url)).get("data")[0].get("users")
+        return (await self.requests_get(url)).get("data")[0].get("users")
```

Comment:
If using `requests_get()` like this then we assume that "data" and "users" will always be in the response. However this is not guaranteed by `requests_get()` as you don't validate that a correct response has been received. Only a 401 error is actually handled in `requests_get()`

Reply:
This issue seems to be in multiple places so I'll have to take more time changing things, yet again something to be added to another issue as unrelated from this pull request (only change was to formatting). After the 3rd I should be able to rewrite TwitchAlert to fix all of these issues